src/query/pipeline/sinks/src/async_mpsc_sink.rs

```diff
@@ -152,6 +152,7 @@ impl<T: AsyncMpscSink + 'static> Processor for AsyncMpscSinker<T> {
     }
 
     #[async_backtrace::framed]
+    #[minitrace::trace(name = "AsyncMpscSinker::async_process")]
```

Comment:
Maybe move it to https://github.com/datafuselabs/databend/blob/main/src/query/pipeline/core/src/processors/processor.rs#L172 is better?

Reply:
if so, we don't know what exact the processor is running.